controllers/tutorial.controller.js

```diff
@@ -10,32 +11,30 @@ const getTutorials = async (req, res) => {
     });
     res.status(200).send(tutorials);
   } catch (error) {
+    logger.error('Internal server error', error);
```

Comment:
Error name should be something we can easily track: `('Couldn't fetch tutorials: ', error)`